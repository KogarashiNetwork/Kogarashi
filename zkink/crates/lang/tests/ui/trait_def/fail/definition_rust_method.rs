use ink_lang as ink;

#[ink::trait_definition]
pub trait TraitDefinition {
    fn rust_method(&self);
}

fn main() {}
