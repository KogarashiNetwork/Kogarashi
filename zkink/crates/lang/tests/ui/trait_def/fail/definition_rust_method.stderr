error: missing #[ink(message)] or #[ink(constructor)] flags on ink! trait method
 --> tests/ui/trait_def/fail/definition_rust_method.rs:5:5
  |
5 |     fn rust_method(&self);
  |     ^^^^^^^^^^^^^^^^^^^^^^
